### PR TITLE
Service vendor nfp for demo

### PR DESCRIPTION
**Pull request by YogeshRajmane**
@njagadish @rajendramachani

### CODE REVIEW

File: devstack/plugin.sh [hunk 1]
```diff
@@ -39,7 +39,6 @@ function nfp_configure_neutron {
     iniset $NEUTRON_CONF group_policy_implicit_policy default_ip_pool "11.0.0.0/8"
     iniset $NEUTRON_CONF group_policy_implicit_policy default_proxy_ip_pool "192.169.0.0/16"
     iniset $NEUTRON_CONF group_policy_implicit_policy default_external_segment_name "default"
-    iniset $NEUTRON_CONF device_lifecycle_drivers drivers "haproxy, vyos"
```
Comment (njagadish) [inline devstack/plugin.sh:42]: Why is this removed? Shouldn't we add nfp vendor to this list rather than removing it?

Comment (YogeshRajmane) [inline devstack/plugin.sh:42]: That option is in neutron.conf and in our code we are not using that, instead of that we are using the option provided
in /etc/nfp_orch.ini file

Comment (rajendramachani) [inline devstack/plugin.sh:42]: This is unused section. Missed while addressing (lifecycle --> orchestration) name changes.

File: gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py [hunk 2]
```diff
@@ -244,7 +244,7 @@ class NFPNodeDriver(driver_base.NodeDriverBase):
         pconst.LOADBALANCER, pconst.FIREWALL, pconst.VPN]
     SUPPORTED_SERVICE_VENDOR_MAPPING = {
         pconst.LOADBALANCER: ["haproxy"],
-        pconst.FIREWALL: ["vyos"],
+        pconst.FIREWALL: ["vyos", "nfp"],
```
Comment (ashutosh-mishra) [inline gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py:247]: Can we have nfp-fw, nfp-lb instead of nfp.

Comment (njagadish) [inline gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py:247]: Makes sense. However, lets do this after summit. Yogesh tested with this change and we have only couple of hours before summit.

**Comment by rajendramachani** [comment]
Raised a new PR on today's daily branch. Added the unaddressed comments in the new PR.
